pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -104,7 +104,11 @@ func (admitter *VMICreateAdmitter) Admit(ar *admissionv1.AdmissionReview) *admis
 		return webhookutils.ToAdmissionResponseError(err)
 	}
 
-	causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("spec"), &vmi.Spec, admitter.ClusterConfig)
+	var causes []metav1.StatusCause
```

Comment:
The missing test is about calling the `ValidateCreation` from the `Admit` flow.
`ValidateCreation` itself is being tested.

To test that the call itself occurs, an additional change is needed here (inject a network admitted stub).
I think we can do it as a follow up, but if you think it is a must, I will try to do the needed changes on this PR.

Reply:
It can be done in a follow-up.
In general, most of the tests moved out from the main validator to a nework specific one have no coverage in the main validator.